cli/cli_fileshare.go

```diff
@@ -203,6 +208,28 @@ func (c *cmd) FileshareAutoCompletePeers(ctx *cli.Context) {
 	}
 }
 
+// FileshareAutoCompleteClear implements bash autocompletion for history clearing
+func (c *cmd) FileshareAutoCompleteClear(ctx *cli.Context) {
+	if ctx.NArg() == 0 {
+		fmt.Println("all\n<time_period>")
+	} else {
```

Comment:
how many arguments can it have?
wouldn't be better to just allow max 2 params `<num>` `<s|m|h|d>` , to keep things simpler? :smile:

Reply:
I tried to keep it compatible with the systemd specification, which allows for example 2h 30min as an equivalent to 150min: https://www.freedesktop.org/software/systemd/man/latest/systemd.time.html